client/components/banner.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import style from '../bootstrap.min.css';
+
+export default class Banner extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div>
+        {/* Template strings (template literals) used for css module format of Bootstrap class names */}
+        {/* See webpack.config.js css-loader options for localIdentName syntax of css output */}
+        <div id="banner" className={ `${style.row} ${style['flex-row']} ${style['justify-content-center']} ${style['col-12']}` }>
```

Comment:
we can make ur life easier and just agree that bootstrap will be loaded in as a global stylesheet.

Reply:
That would be easier, but isn't css modules syntax a requirement?

Reply:
i would say for module specific things aka our service widgets aka non library based css aka your own created css. if you think about it the scaffold itself has css too right? generally there will alway be some global stylings, and if we're all sharing bootstrap styles no need to individually scope it.

Reply:
That's a lot less typing, so 🙏for that